strawberryfields/gbs/vibronic.py

```diff
@@ -194,26 +194,28 @@ def gbs_params(
     return t, U1, np.log(s), U2, alpha
 
 
-def energies(samples: list, wp: np.ndarray) -> Union[list, float]:
+def energies(samples: list, w: np.ndarray, wp: np.ndarray) -> Union[list, float]:
     r"""Computes the energy of each GBS sample in units of :math:`\text{cm}^{-1}`.
 
     **Example usage:**
 
-    >>> samples = [[1, 1, 0], [1, 0, 2]]
+    >>> samples = [[1, 1, 0, 0, 0, 0], [1, 2, 0, 0, 1, 1]]
+    >>> w  = np.array([300.0, 200.0, 100.0])
     >>> wp = np.array([700.0, 600.0, 500.0])
     >>> energies(samples, wp)
-    [1300.0, 1700.0]
+    [1300.0, 1600.0]
 
     Args:
         samples (list[list[int]] or list[int]): a list of samples from GBS, or alternatively a
             single sample
```

Comment:
In what situations do we expect the input to be a single sample?

Reply:
Just wanted to make it general in case a user, for any reasons, wants the energy of a single sample.